test/pageobjects/register.ts

```diff
@@ -18,6 +18,44 @@ class Registration extends Page {
   get masterPasswordInput() {
     return $("#register-form_input_master-password");
   }
+
+  async masterPasswordBitError() {
+    let masterPasswordInput = await $("#register-form_input_master-password");
+    let parentElement = await masterPasswordInput.parentElement();
+    let bitErrorElement = await parentElement.$("div[id*=bit-error]");
+    return bitErrorElement;
+  }
+
+  async confirmMasterPasswordBitError() {
+    let confirmMasterPasswordInput = await $(
+      "#register-form_input_confirm-master-password"
+    );
+    let parentElement = await confirmMasterPasswordInput.parentElement();
+    let bitErrorElement = await parentElement.$("div[id*=bit-error]");
+    return bitErrorElement;
+  }
+  get errorSummary() {
+    return $(
+      "bit-error-summary.tw-block.tw-text-danger.tw-mt-2.ng-touched.ng-dirty.ng-invalid"
+    );
+  }
+
+  get dialogContainer() {
+    return $("#cdk-dialog-container");
+  }
+
+  get dialogHeader() {
+    return $(
+      "div.tw-flex.tw-flex-col.tw-items-center.tw-gap-2.tw-px-4.tw-pt-4.tw-text-center.ng-tns-c3905586946-0"
+    );
+  }
+
+  get noButton() {
+    return $(
```

Comment:
Not possible to read this selector, are you sure you need it to be that long?

Reply:
selector has been shortened